File: go/main/test/alphabetic_shift_test.go
package main

import (
	"reflect"
	"testing"
)

func TestAlphabeticShift(t *testing.T) {
	data := "crazy"
	got := AlphabeticShift(data)
	want := "dsbaz"
	if !reflect.DeepEqual(got, want) {
		t.Errorf("AlphabeticShift(#{data}) = #{got} want #{want}")
	}
}

File: php/tests/AlphabeticShiftTest.php
<?php

use PHPUnit\Framework\TestCase;
require "src/alphabetic_shift.php";

final class AlphabeticShiftTest extends TestCase
{
    public function testAlphabeticShift()
    {
        $data = "crazy";
        $actual = alphabetic_shift($data);
        $expected = "dsbaz";
        $this->assertEquals($expected, $actual);
  }
}
